apps/scan/backend/src/vx_interpreter.test.ts

```diff
@@ -36,15 +38,20 @@ import { Interpreter, sheetRequiresAdjudication } from './vx_interpreter';
 const interpreterOutputPath = join(__dirname, '..', 'test-output-dir/');
 emptyDirSync(interpreterOutputPath);
 
-jest.mock('@votingworks/ballot-encoder', () => {
+// mock SKIP_ELECTION_HASH_CHECK to allow us to use old ballot image fixtures
+const featureFlagMock = getFeatureFlagMock();
+jest.mock('@votingworks/utils', () => {
   return {
-    ...jest.requireActual('@votingworks/ballot-encoder'),
-    // to allow changing election definitions without changing the image fixtures
-    // TODO: generate image fixtures from election definitions more easily
-    // this election hash is for the MS demo fixture images
-    sliceElectionHash: () => 'b0260b4e9d492dab3813',
+    ...jest.requireActual('@votingworks/utils'),
+    isFeatureFlagEnabled: (flag: BooleanEnvironmentVariableName) =>
+      featureFlagMock.isEnabled(flag),
```

Comment:
This boilerplate is probably my least favorite part of this approach. Would it be possible to write it like this:

```ts
jest.mock('@votingworks/utils', () => {
  return {
    ...jest.requireActual('@votingworks/utils'),
    ...featureFlagMock.exports,
  }
})
```

Or something like that. Just thinking of ways to reduce this

Reply:
I would love to be able to reduce this. At first I just had this: 
```ts
  return {
    ...jest.requireActual('@votingworks/utils'),
    isFeatureFlagEnabled: featureFlagMock.isEnabled,
  }
```
I like that it's slightly less boilerplate, but it won't work because the `jest.mock` is hoisted and `featureFlagMock` is not defined at the time that its property would be accessed.

I'm trying to wrap my head around your `.exports` idea. On of the reasons to actually instantiate something (to have a `getFeatureFlagMock` rather than a singleton mock) is because I was concerned about `jest` files being run in parallel, and feature flag mocks interfering with one another. But if they're being run in parallel, can they interfere with one another if they share a dependency? Now that I think about it, I think that `jest` re-imports dependencies for every test suite so it would actually be safe to have a singleton pattern, then you could spread in like this. Thoughts?

Reply:
Ugh I forgot about the hoisting. That's annoying. I think the less shared state across modules, the less likely it is for something confusing to happen, so I'm inclined to just keep it as you have it here.

We could also explore avoiding the hoisting. Looks like `jest.doMock` is the way to do that: https://jestjs.io/docs/jest-object#jestdomockmodulename-factory-options

Reply:
I've tried to get `jest.doMock` to work in our codebase several times and never gotten it to work. There are some conditions around it I don't fully understand. If you ever figure it out, that would be excellent!